### PR TITLE
update message and onclick notification on post removed

### PR DESCRIPTION
**Pull request by kriptonhaz**
# Title
- [x] JIRA Issue ID (preferably only ONE parent issue, put subtask IDs in the Description)
- [x] Multiple JIRA Issue IDs? Are you sure your PR isn't too big? Can you rebase your commits to new branches so there's only 1 issue ID?
- [x] Is your title relatable to the JIRA issue?
- [x] Title <= 100 characters

# Description

Notification on post removed detected on the message from API and differentiate either it has includes 'approved' string or not and put a handle notification to detect wether the href props has a '@' character or not to either push the route or send an email to official support.

# How Has This Been Tested?

Please describe the tests that you ran to verify your changes. Please also note any relevant details for your test configuration.

- [x] Builds and runs on Chrome Desktop
- [ ] Builds and runs on Chrome Responsive
- [ ] Unit tests?
- [ ] Integration tests?
- [ ] End-to-end (E2E) tests?

# Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [x] Any dependent changes have been merged and published in downstream modules
- [x] Have you added yourself as the Assignee?
- [x] Have you added 1 or more leads as the Reviewer?
- [x] Have you chosen 1-2 of the following labels?
  - Feature
  - Bug
  - Refactor
  - Test
  - Documentation

### CODE REVIEW

File: src/components/Notifications/hooks/use-notification-list.hook.tsx [hunk 1]
```diff
@@ -370,7 +372,9 @@ export const useNotificationList = (
               </div>
             ),
             createdAt: notification.createdAt,
-            href: `support@myriad.social`,
+            href: notification.message.includes('approved')
+              ? ''
+              : `support@myriad.social?subject=Complain post take down!`,
```
Comment (irmannmal) [inline src/components/Notifications/hooks/use-notification-list.hook.tsx:377]: please use env config to get mail support

https://github.com/myriadsocial/myriad-web/blob/12d02c06683f46f72306ed2f211664b6394a0c46/next.config.js#L36

File: src/components/Notifications/hooks/use-notification-list.hook.tsx [hunk 2]
```diff
@@ -359,7 +359,9 @@ export const useNotificationList = (
             read: notification.read,
             user: 'Post removed',
             avatar: notification.fromUserId.profilePictureURL,
-            description: 'Your post has been removed due to breaking our community guideline',
+            description: notification.message.includes('approved')
+              ? 'Report has been approved'
```
Comment (irmannmal) [inline src/components/Notifications/hooks/use-notification-list.hook.tsx:363]: please confirm this wording to PM or QA

File: src/components/Notifications/Notifications.tsx [hunk 3]
```diff
@@ -78,7 +78,11 @@ export const Notifications: React.FC<NotificationsProps> = props => {
 
   const handleReadNotification = (notification: NotificationList) => () => {
     onMarkItemAsRead(notification.id, () => {
-      router.push(notification.href);
+      if (notification.href.includes('@')) {
```
Comment (irmannmal) [inline src/components/Notifications/Notifications.tsx:81]: sort its condition as soon as possible, and safety null

maybe like this

```
if (!notification.href && notification.href.trim().length === 0) return

if (notification.href.includes('@')) {
    window.open(`mailto:${notification.href}`, '_blank');
 }
```

**Review by irmannmal — CHANGES_REQUESTED** [review]

File: src/components/Notifications/hooks/use-notification-list.hook.tsx (hunk 2 above)
Comment (kriptonhaz) [reply src/components/Notifications/hooks/use-notification-list.hook.tsx:363]: already confirm with yogi

**Review by irmannmal — APPROVED** [review]

**Pull request merged**